{{cookiecutter.project_dirname}}/docker-compose.yaml

```diff
@@ -0,0 +1,39 @@
+version: "3.4"
+
+services:
+
+  backend:
+    build:
+      args:
+        USER: ${USER}
+      context: .
+      target: ${BACKEND_TARGET:-local}
```

Comment:
Questa riga potrebbe essere utile riportala anche nell'omologo file dell'orchestratore?

Reply:
No, we have nginx to expose the service there.